__tests__/get.test.js

```diff
@@ -0,0 +1,90 @@
+const app = require('../server.js')
+const supertest = require('supertest')
+const request = supertest(app)
+
+it('tests the endpoint for the entire list', async() => {
+    const magicItems = [{
```

Comment:
nice! I might suggest having this in a separate data file and importing it, but still works!